src/order-and-subscriptions/OrderAndSubscriptionsPage.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect } from 'react';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+
+import Subscriptions from '../subscriptions';
+import Payments from '../order-history';
+
+import messages from './OrderAndSubscriptionsPage.messages';
+
+const OrderAndSubscriptionsPage = ({ intl }) => {
+  const isB2CSubsEnabled = true;
+
+  const message = (id) => intl.formatMessage(messages[id]);
+
+  useEffect(() => {
+    if (isB2CSubsEnabled) {
+      document.title = 'Orders and Subscriptions | edX';
+    }
+  }, [isB2CSubsEnabled]);
+
+  if (!isB2CSubsEnabled) {
+    return (
+      <div className="page__order-and-subscriptions container-fluid py-5">
+        <Payments isB2CSubsEnabled={false} />
+      </div>
+    );
+  }
+
+  return (
+    <div className="page__order-and-subscriptions container-fluid py-4.5">
+      <div className="section">
+        <h1 className="text-primary-700">
+          {message('ecommerce.order.history.main.heading')}
```

Comment:
Please prefer to use `<FormattedMessage/>` here. It will make JSX more readable and we should prefer `intl.formatMessage` where we need to conditionally render, or inject things in it or to form complex messages.

Reply:
sure